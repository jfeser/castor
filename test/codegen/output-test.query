select([1 as a, 100 as b, 1.0 as c, 0.0001 as d, 1.0001 as f, "this is a test" as g, "test with, commas" as h, date("1994-01-04") as i, true as j, false as k], ascalar(0 as l))