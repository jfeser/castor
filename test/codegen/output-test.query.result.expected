1|100|1.000000|0.000100|1.000100|this is a test|test with, commas|1994-01-04|t|f
